conf/messages.en

```diff
@@ -38,6 +38,10 @@ error.non_numeric = Give a value using only numbers
 error.number = Please enter a valid number
 error.required = Please enter a value
 error.summary.title = There is a problem
+error.greaterThanOneBillion = Enter an amount lower than 1000000000
```

Comment:
This would mean 1000000000 is not allowed. Actually anything greater than 1000000000 is not allowed

Reply:
Yes. I would suggest something like "Enter an amount lower than or equal to 1000000000" Who should I confirm this with? Darryl?

Reply:
Yes, with Darryl